library/Kubernetes/Web/JobDetail.php

```diff
@@ -77,29 +78,42 @@ protected function assemble(): void
             ])),
             new Labels($this->job->label),
             new Annotations($this->job->annotation),
-            new JobConditions($this->job),
-            new HtmlElement(
+            new JobConditions($this->job)
+        );
+
+        if (Auth::getInstance()->hasPermission(Auth::SHOW_PODS)) {
+            Auth::getInstance()->applyRestrictions($this->job->pod->with(['node']), Auth::SHOW_PODS);
+
+            $this->addHtml(new HtmlElement(
                 'section',
                 null,
                 new HtmlElement('h2', null, new Text($this->translate('Pods'))),
                 new PodList($this->job->pod->with(['node']))
-            ),
-            new HtmlElement(
+            ));
+        }
+
+        if (Auth::getInstance()->hasPermission(Auth::SHOW_EVENTS)) {
+            $events = Event::on(Database::connection())
```

Comment:
I think this can be harmonized with the other event queries, i.e.
```php
            $events = Event::on(Database::connection())
                ->filter(Filter::equal('referent_uuid', $this->job->uuid));
```